pydocstringformatter/configuration/arguments_manager.py

```diff
@@ -108,6 +108,13 @@ def _register_arguments(self, version: str) -> None:
             ),
         )
 
+        self.configuration_group.add_argument(
+            "--max-line-length",
+            action="store_const",
```

Comment:
```suggestion
            action="store",
```

See `argparse` [documentation](https://docs.python.org/3/library/argparse.html#action). `store_const` actually stores one value `88` in this case and doesn't allow specifying another value. `store` should work for us!

Reply:
Where do I set it to default 88 then?

Reply:
```suggestion
            action="store",
            default=88,
            type=int,
```

See the example in the argparse docs:
```python
>>> parser = argparse.ArgumentParser()
>>> parser.add_argument('--foo', default=42)
>>> parser.parse_args(['--foo', '2'])
Namespace(foo='2')
>>> parser.parse_args([])
Namespace(foo=42)
```